wire-library/wire-schema/build.gradle.kts

```diff
@@ -74,3 +74,6 @@ configure<MavenPublishBaseExtension> {
     KotlinMultiplatform(javadocJar = Dokka("dokkaGfm"))
   )
 }
+
+tasks.withType<ProcessResources> { duplicatesStrategy = DuplicatesStrategy.INCLUDE}
```

Comment:
I wonder if this is strictly necessary... I saw that the build failed without it, but we're not changing anything here that would trigger this.